x/accesscontrol/module.go

```diff
@@ -139,7 +139,7 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 }
 
 // ConsensusVersion implements AppModule/ConsensusVersion.
-func (AppModule) ConsensusVersion() uint64 { return 2 }
+func (AppModule) ConsensusVersion() uint64 { return 1 }
```

Comment:
acl module should start with 1 since there are no registered migrations from 1 -> 2